tests/kernel/mbox/mbox_api/src/test_mbox_api.c

```diff
@@ -185,19 +368,181 @@ static void tmbox_get(struct k_mbox *pmbox)
 		zassert_true(k_mbox_get(pmbox, &mmsg, NULL, K_FOREVER) == 0,
 			     NULL);
 		zassert_true(k_mbox_data_block_get
-				     (&mmsg, &mpoolrx, &rxblock, K_FOREVER) == 0, NULL);
```

Comment:
why change this line?

Reply:
Was the only line in the file which exceeded beyond 80 character. Hence fixed it.